docs/release_notes/release-2.3.md

```diff
@@ -1,5 +1,20 @@
 # StarRocks version 2.3
 
+## 2.3.8
+
+Release date: February 2, 2023
+
+### Bug fixes
+
+The following bugs are fixed:
+
+- When resources are released after a large query finishes, there is a low probability that other queries are slowed down. This issue is more likely to occur if resource groups are enabled or the large query ends unexpectedly. [#16454](https://github.com/StarRocks/starrocks/pull/16454) [#16602](https://github.com/StarRocks/starrocks/pull/16602)
+- For a primary key table, if a replica's metadata version falls behind, StarRocks performs incremental clone to clone the missing metadata from other replicas to this replica. In this process, StarRocks pulls a large number of versions of metadata, and if too many versions of metadata accumulate without timely GC, excessive memory may be consumed and consequently the BEs may encounter OOM exceptions. [#15935](https://github.com/StarRocks/starrocks/pull/15935)
+- If an FE sends an occasional heartbeat to a BE, and the heartbeat connection times out, the FE considers the BE unavailable, leading to transaction failures on the BE. [# 16386](https://github.com/StarRocks/starrocks/pull/16386)
+- When you use a StarRocks external table to load data between StarRocks clusters, if the source StarRocks cluster is in an earlier version and the target StarRocks cluster is in a later version (2.2.8 ~ 2.2.11, 2.3.4 ~ 2.3.7, 2.4.1 or 2.4.2), the data loading fails. [#16173](https://github.com/StarRocks/starrocks/pull/16173)
+- BEs crash when multiple queries run concurrently and memory usage is relatively high. [#16047](https://github.com/StarRocks/starrocks/pull/16047)
+- When a table enables dynamic partitioning and some partitions are dynamically deleted, if you execute TRUNCATE TABLE, an error NullPointerException returns. Meanwhile, if data load into the table, FE crashes and can not restart. [#16822](https://github.com/StarRocks/starrocks/pull/16822)
+
```

Comment:
When dynamic partitioning is enabled for a table and some partitions are dynamically deleted, if you execute TRUNCATE TABLE, an error NullPointerException is returned. Meanwhile, if data is being loaded into the table, the FE crashes and cannot restart.